eng/build.yml

```diff
@@ -17,161 +17,165 @@ parameters:
   # Publish using pipelines
   enablePublishUsingPipelines: true
 
-phases:
-- template: /eng/common/templates/job/job.yml
-  parameters:
-    ${{ if parameters.pgoInstrument }}:
-      name: PGO_${{ parameters.agentOs }}
-    ${{ if not(parameters.pgoInstrument) }}:
-      name: ${{ parameters.agentOs }}
-    timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
-    ${{ if eq(parameters.agentOs, 'Windows_NT') }}:
-      enableMicrobuild: true
-    enablePublishBuildAssets: true
-    # Don't use built-in arcade logic for publishing artifacts and test results, as we manually do this and give better names
-    #enablePublishBuildArtifacts: true
-    #enablePublishTestResults: true
-    enablePublishUsingPipelines: ${{parameters.enablePublishUsingPipelines}}
-    enableTelemetry: true
-    helixRepo: dotnet/installer
-    pool: ${{ parameters.pool }}
-    ${{ if ne(parameters.strategy, '') }}:
-      strategy: ${{ parameters.strategy }}
-    workspace:
-      clean: all
-
-    variables:
-      - _PgoInstrument: ''
+stages:
```

Comment:
I took a look at the code and this change doesn't need to be done, this is already on an Stage called "Build" https://github.com/dotnet/installer/blob/8288ae2e6d7b990e2cd29ce1180eabfda7b96599/.vsts-ci.yml#L37

This is because this pipeline is called from the other one that already have the stage defined. I also double checked this information looking at one of the build of this pipeline and the information is in there, so you don't need to do any of this. 

The only thing left do be done is publish the file. I will do that on [vsts-ci.yml](https://github.com/dotnet/installer/blob/8288ae2e6d7b990e2cd29ce1180eabfda7b96599/.vsts-ci.yml) instead under jobs
```
stages:
- stage: build
   jobs:
      - job: Publish_Build_Configuration
        steps:
          - publish: $(Build.SourcesDirectory)\eng\BuildConfiguration
            artifact: BuildConfiguration
            displayName: Publish Build Config
      - template: /eng/build.yml
```